### PR TITLE
Improved the documentation on using Kubernetes or OpenShift.

### PR DESCRIPTION
**Pull request by deanwampler**
No changes to KillrWeather, as Boris is working a PR for it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,31 +24,53 @@ The applications are organized in folders with each of them containing details o
...
+
+### Kubernetes and OpenShift
+
+For both platforms, [Helm](https://helm.sh/) is used to install the sample apps. Make sure the following prerequisites are installed first:
+
+* The Fast Data Platform cluster is up and running
+* The user is authenticated to the cluster
+* The cluster has sufficient resources to host the applications
+* The prerequisites mentioned above are installed.
+
```
Comment (blublinsky) [inline README.md]: We do provide helm chart for all prerequisites (except for Spark - no initial install is required) here https://github.com/lightbend/fdp-sample-applications/tree/openshift/supportingcharts - for some reason it is not merged

File: apps/kstream/README.md [hunk 2]
```diff
@@ -348,11 +348,11 @@ $ helm install --name kstream ./helm
...
 
 ### Using the traefik Ingress Controller
 
-`kstream` application publishes a REST end point and allows users to query on various application state information. In Kubernetes this can be done in a way so that the user gets one fixed end point irrespective of the ways the underlying PODs are distributed across the cluster. Even in case of PODs getting restarted on a different node, it would be great to still have the fixed end point. We use the traefik ingress controller for this.
+`kstream` application publishes a REST end point and allows users to query on various application state information. In Kubernetes or OpenShift this can be done in a way so that the user gets one fixed end point irrespective of the ways the underlying PODs are distributed across the cluster. Even in case of PODs getting restarted on a different node, it would be great to still have the fixed end point. We use the traefik ingress controller for this.
 
 > Ensure that the traefik ingress controller is installed in the cluster. Check that the dashboard is accessible through some end points like http://<public agent IP>:9901/dashboard/
```
Comment (blublinsky) [inline apps/kstream/README.md]: I think we want to remove this and write something like:
To expose service on kubernetes use Ingress (on vanila kubernetes) or Route (on OpenShift)

Comment (deanwampler) [reply apps/kstream/README.md]: Okay. I just pushed a small change to the text here. See what you think.

Comment (blublinsky) [reply apps/kstream/README.md]: Looks good, but I would probably remove traefik completely. It was a hack that we did for k8 on DC/OS

**Comment by deanwampler** [comment]
There are a few paragraphs here that discuss traefik. What should the
replacement text be? I'll insert it, then push an update.

**Comment by deanwampler** [comment]
Merging This and adding a task for @blublinsky to fix the traefik wording when convenient.

**Pull request merged**